src/UI/examples/Panel/Secondary/Legacy/with_viewcontrollers.php

```diff
@@ -27,7 +27,7 @@ function with_viewcontrollers()
     $url = $DIC->http()->request()->getRequestTarget();
 
     $parameter_name = 'page';
-    $current_page = (int) @$_GET[$parameter_name];
+    $current_page = (int) (array_key_exists($parameter_name, $_GET) ? $_GET[$parameter_name] : 0);
```

Comment:
Would we want to use this construct [here](https://github.com/ILIAS-eLearning/ILIAS/pull/3222/files#diff-284fa4639eb987095447333cc1447d1326c33b9bb0c963017bc78006975c0a05R39) and [there](https://github.com/ILIAS-eLearning/ILIAS/pull/3222/files#diff-bde9e13b5cad94ebe56ee2ac3bc34ce532b0a831ee455d228a472cb01061d442R27)? Or could we just use `$_GET[$parameter_name] ?? 0`? This would then also need to be changed [here](https://github.com/ILIAS-eLearning/ILIAS/pull/3222/files#diff-063e35698485577d1146c3bf20dddb1bd6ccbb669ab6118b7c435accf5f66942R51), [here](https://github.com/ILIAS-eLearning/ILIAS/pull/3222/files#diff-a56c7e4d9ce3f7427a3a5533c02d563ac446d4d55832da3f727e581c45418d4cR30), [here](https://github.com/ILIAS-eLearning/ILIAS/pull/3222/files#diff-671784fa572ee3182b855bcd815191f10e2f69bf586819c8905a6692e2c14699R13), [here](https://github.com/ILIAS-eLearning/ILIAS/pull/3222/files#diff-26d1e677c6db219f15bda44f9dd4e51f9c9286757f75d896c2c55e87178ee468R13) and [there](https://github.com/ILIAS-eLearning/ILIAS/pull/3222/files#diff-5b772ef5bd99a0d70d86b6a949a6d381a4ec0597bd3ca59738579d7446397c99R14).

Reply:
I used the array_key_exists pattern, with the other one, we would IMO still run into the issue of not defined array key indices. I changed the linked place as well.

Reply:
@Amstutz and me almost agreed to move to the new `??` sometimes soon =)